python-requirements.txt

```diff
@@ -11,7 +11,6 @@ attrs==21.4.0
 Babel==2.10.1
 bcp47==0.0.4
 beautifulsoup4==4.12.2
-bioregistry==0.6.99
```

Comment:
@joeflack4 why is the `curies` package not included in the python-requirements.txt file in addition to the python-requirements-unlocked.txt file?

Reply:
Ah. I don't maintain this file anymore. It doesn't say it (maybe it should), but this requirements file applies to Intel / x86 architecture.

After I got my new computer last December, I'm now on ARM64 Apple Silicon. I wasn't able to install via this requirements file. `pandas` (`numpy`) may have been the only thing that wouldn't install. Since then, I created and started maintaining a separate `python-requirements-apple-silicon.txt` (it should really say `-arm64`).

Even though I can't `pip freeze > python-requirements.txt`, I could manually insert the `curies` line in there.

Perhaps we should make an issue about the best way to handle these multiple `requirements.txt` files. Perhaps we should just drop `requirements.txt` since nobody is maintaining it anymore, and rename the other one to `-arm64.txt`, and be done with it for now.I've never seen a Python project before with multiples of them before, though I have seen the distinction in more sophisticated setups e.g. Poetry.

- [ ] @twhetzel Decide: (a) drop `requirements.txt`, (b) manually add `curies` to this file, (c) something else.

Reply:
The `curies` package is listed in this file. I missed that earlier.